### ISSUE TITLE
npm WARN cannot run in wd nwm@1.0.1 node-gyp rebuild

### ISSUE THREAD

**Issue by torleif** [mixu/nwm#41]
Tried to install on Debian (6 stable)

I receive the error `npm WARN cannot run in wd nwm@1.0.1 node-gyp rebuild`

tried running as root, and forcing it to run via npm. Still no luck.

**Comment by Ineentho** [comment]
I had the problem as well. The thing is I'm using iojs (I suspect it's the same thing with node .12). Most things are forwards compatible so I didn't really care of the requirements you listed (.6, .8 or .10), but seems like .10 is required.

In short: Using node .10 did the trick for me

**Comment by AKRFranko** [comment]
If it helps, I've had similar issues on one of my Arch Linux boxes because of confusion between multiple python versions and binaries. IIRC all these issues went away for me when I did these things:

1. Updated my system package manager and system packages realted to python.
2. Made certain that my python version is properly set up in the shell regardless of node.
3. Told npm which python to use as mentionned here  https://github.com/TooTallNate/node-gyp ie: `npm config set python /path/to/executable/python2.7`

FWIW, maybe do this and then also what @mauvm said https://github.com/mixu/nwm/issues/41#issuecomment-66833641
